tools/InterfaceGenerator/generator/generators/PolicyTypes.py

```diff
@@ -0,0 +1,575 @@
+"""SmartFactory code generator base.
+
+Base of code generator for SmartFactory that provides SmartSchema object in
+accordance with given internal model.
+
+"""
+# pylint: disable=W0402
+# pylint: disable=C0302
+import codecs
+import collections
+import os
+import string
+import uuid
+
+from generator import Model
+
+
+class GenerateError(Exception):
+
+    """Generate error.
+
+    This exception is raised when generator is unable to create
+    output from given model.
+
+    """
+
+    pass
+
+def to_camel_case(snake_str):
+    components = snake_str.split('_')
+    return components[0].lower() + "".join(x.title() for x in components[1:])
+
+class CodeGenerator(object):
```

Comment:
This is not ok to be included in this pr. Please separate the policy generator into a separate into a seperate PR that we can review. These changes will break other 6.0 new RPC features being merged into develop.

Reply:
This functionality was removed and PR was updated